packages/node_modules/@webex/react-component-utils/src/activity.js

```diff
@@ -10,11 +12,23 @@ import {isImage} from './files';
  * @param {object} conversation
  * @param {object} activityObject
  * @param {any} actor
+ * @param {boolean} [sanitize = false]
  * @returns {object}
  */
-export function constructActivity(conversation, activityObject, actor) {
+export function constructActivity(conversation, activityObject, actor, sanitize = false) {
   const clientTempId = `sdk-widget-${uuid.v4()}`;
 
+  const modifiedActivityObject = Object.assign({}, activityObject);
+
+  if (sanitize) {
+    if (modifiedActivityObject.displayName) {
+      modifiedActivityObject.displayName = escapeSync(activityObject.displayName);
```

Comment:
Don't do it for `displayName`

Reply:
Why? It will display one or the other if there is no content

Reply:
We will render either https://github.com/adamweeks/react-ciscospark/blob/6a3a8a037cd4c7004305aff69405c55cec03953c/packages/node_modules/@webex/react-component-activity-text/src/index.js#L29 so we need to sanitize both before sending to dom

Reply:
The other spot where sanitization happens in the widgets (done by the adaptive cards team), they also modify the `displayName`
https://github.com/webex/react-widgets/blob/master/packages/node_modules/@webex/redux-module-activity/src/helpers.js#L101

Reply:
We should not be displaying `displayName` like that. That is a bug.
If `content` is not there, you use `displayName` but you don't set it like it is HTML. One should display it like a string of characters.

Reply:
@mwegman I added another commit that fixes the text render component to not html render `displayName` and have removed the sanitization of that property.